src/util/addBoxPlot.js

```diff
@@ -1,30 +1,20 @@
-export default function addBoxplot
-    (svg
-    ,results
-    ,height
-    ,width
-    ,domain
-    ,boxPlotWidth
-    ,boxColor
-    ,boxInsideColor
-    ,format = '.2f'
-    ,horizontal = true) {
+export default function addBoxplot(chart, group
```

Comment:
The group object has a bunch of items of interest, most importantly:

* The x-axis value at which to plot the box plot
* The group value with which to offset the the box plot
* The nested group data

Rather than pass each of those to the function I thought it would be nicer to just wrap 'em up in a single object.